vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -177,33 +172,36 @@ func (d *VolumeDriver) MountVolume(name string, fstype string, id string, isRead
 		return mountpoint, err
 	}
 
-	watcher, skipInotify := fs.DevAttachWaitPrep(name, watchPath)
-
-	// Have ESX attach the disk
-	dev, err := d.ops.Attach(name, nil)
-	if err != nil {
-		return mountpoint, err
+	watchCtx, errWatch := fs.DevAttachWaitPrep()
```

Comment:
is the var rename really needed ? I understand it'slinux dependent. but it was semantically valid and I am not sure what errWatch means. Or is it standard "err" from a go func ? then name it 'err' :-)